### ISSUE TITLE
Dates inside datepicker are not accessible using voiceover in ios devices.

### ISSUE THREAD

**Issue by aayugupta20** [react-native-datetimepicker/datetimepicker#968]
While using date picker in application, it is found that focus does not reach to individual dates in the calendar while using voiceover hence violating WCAG 2.1.1 criteria. Providing the sample code as how we are rendering Datepicker-

<DateTimePicker
    value={value ? new Date(value) : new Date()}
    mode="date"
    display={Platform.OS === 'ios' ? 'inline' : 'default'}
    minimumDate={minDate && new Date(minDate)}
    maximumDate={maxDate && new Date(maxDate)}
    onChange={(e, val) => {
      handlePickerChange(val);
    }}
  />
